### PR TITLE
[feature/RT-1128] to dev

### PR DESCRIPTION
**Pull request by jdelrioiriusrisk**

### CODE REVIEW

**Comment by rramonpa** [comment]
Name of PR changed following our guidelines

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        allowInsecureProtocol true
+        url "http://localhost:9092/repository/iriusrisk-central/"
+    }
+}
+
+dependencies {
+    implementation "org.apache.httpcomponents:httpclient:4.5.13"
+    implementation "org.apache.httpcomponents:httpmime:4.2.5"
+    implementation "net.sf.json-lib:json-lib:2.4:jdk15"
+    implementation 'joda-time:joda-time:2.3'
```
Comment (rramonpa) [inline build.gradle]: Minor change: I prefer to use the same double quote here, like other dependencies.

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (rramonpa) [inline gradle.properties]: For this file, the comments is preferable to use the character '#'

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,6 @@
 # Custom rules
 
+gradle/
```
Comment (rramonpa) [inline .gitignore]: Why do we are ignoring this folder? We need the wrapper to be a portable project.

File: pom_old.xml
Comment (rramonpa) [inline pom_old.xml:1]: We should remove this file if it is not necessary.

File: gradle.properties [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+* Copyright (c) 2012-2023 IriusRisk SL. All rights reserved
+*/
+
+version=1.2.0-SNAPSHOT
```
Comment (jdelrioiriusrisk) [inline gradle.properties]: wich version should we have here? we keep 1.2.0-SNAPSHOT as there is no change in code?

Comment (rramonpa) [reply gradle.properties]: Yes, we keep version 1.2.0-SNAPSHOT

**Review by rramonpa — APPROVED** [review]
Review done

**Pull request merged**